frontend/android/app/src/main/java/elte/moneyshare/util/TimeExtensions.kt

```diff
@@ -0,0 +1,27 @@
+package elte.moneyshare.util
+
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.TimeZone
+import java.util.Calendar
+
+class TimeExtensions {
```

Comment:
rename to DateExtensions, and don't need to create class, simply you can create extensions methods to any type.
like this:
fun formatDate(calendar: Calendar): String = DateFormat.getDateInstance().format(calendar.time)
->
fun Calendar.formatDate(): String = DateFormat.getDateInstance().format(time)
"this" (this.time) will be the object on called this method, dont need plus paramter and the static class to use it